lib/utils/nginx_parser.rb

```diff
@@ -22,7 +22,7 @@ class NginxParser < Parslet::Parser
   }
 
   rule(:identifier) {
-    (match('[a-zA-Z]') >> match('[a-zA-Z0-9_]').repeat).as(:identifier) >> space >> space.repeat
+    (match('[a-zA-Z]') >> match('\S').repeat).as(:identifier) >> space >> space.repeat
```

Comment:
Tiny explanation: having `/` in these lines messed up the parser. I tracked down what identifiers potentially look like and it's really anything but a space. If we get more details we can specify it closer.